src/flint/fq.jl

```diff
@@ -599,15 +599,14 @@ sparse, irreducible polynomial is used. The generator of the field is
 guaranteed to be a multiplicative generator only if the field is generated by
 a Conway polynomial. We require the degree to be positive.
 """
-function FlintFiniteField(char::IntegerUnion, deg::Int, s::AbstractString; cached = true)
-   char = fmpz(char)
+function FlintFiniteField(char::fmpz, deg::Int, s::Union{AbstractString,Symbol} = :o; cached = true)
```

Comment:
I've replaced IntegerUnion by fmpz here because if the first argument is `Int`, then the constructor from `fq_nmod.jl` will always be called over this one, as it is more specific